generate_workspace/BUILD

```diff
@@ -1,9 +0,0 @@
-load('//:generate_workspace.bzl', 'generated_java_libraries')
```

Comment:
Removed per recommendations. However, with this removed, where/when do we define the `java_binary`